framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/UiNotificationOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Volo.Abp.AspNetCore.Components.WebAssembly
+{
+    /// <summary>
+    /// Options to override notification appearance.
+    /// </summary>
+    public class UiNotificationOptions
+    {
+        /// <summary>
+        /// Custom text for the Ok button.
+        /// </summary>
+        public string OkButtonText { get; set; }
```

Comment:
It should be a `ILocalizableString`

Reply:
Can you explain what is ILocalizableString and why it's better that string?

Reply:
If you make it [ILocalizableString](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.Localization.Abstractions/Volo/Abp/Localization/ILocalizableString.cs) (and handle while showing it), the text is automatically localized based on the current language. Otherwise, the value will be same in every language. What you need to do is;

1. Make this `public ILocalizableString OkButtonText { get; set; }`

In this way, I can set it like `new LocalizableString(typeof(MyResource), "Key")`.

2. When you need to show this, use `var currentValue = OkButtonText.Localize(StringLocalizerFactory)`. Inject and pass `IStringLocalizerFactory` here, it knows how to localize it.

Reply:
Ok, yeah that is better indeed. Thanks

Reply:
Converted to ILocalizableString